### ISSUE TITLE
Link from homebridge is broken

### ISSUE THREAD

**Issue by mlittle408** [balansse/homebridge-vivint#49]
This link redirects to a 404:
https://github.com/@balansse/homebridge-vivint#readme

This one works:
https://github.com/balansse/homebridge-vivint#readme

I guess Github made a change to their links?

**Comment by jgrimard** [comment]
No, that was my fault. I made a typo. This has already been fixed via a pull request a few days ago. @balansse just needs to publish the changes to NPM.

**Comment by balansse** [comment]
Fixed in 1.6.1
